pyart/aux_io/arm_vpt.py

```diff
@@ -0,0 +1,351 @@
+        'long_name': 'Target angle for sweep',
+        'standard_name': 'target_fixed_angle',
+        'units': 'degrees',
+    }
+    sweep_start_ray_index = {
+        'data': np.array([0], dtype=np.int32),
+        'long_name': 'Index of first ray in sweep, 0-based',
+        'standard_name': 'index_of_first_ray_in_sweep',
+        'units': 'count',
+    }
+    sweep_end_ray_index = {
+        'data': np.array([ncvars['time'].size - 1], dtype=np.int32),
+        'long_name': 'Index of last ray in sweep, 0-based',
+        'standard_name': 'index_of_last_ray_in_sweep',
+        'units': 'count',
+    }
+
+    #sweep_number = _ncvar_to_dict(ncvars['sweep_number'])
+    #sweep_mode = _ncvar_to_dict(ncvars['sweep_mode'])
+    #fixed_angle = _ncvar_to_dict(ncvars['fixed_angle'])
+    #sweep_start_ray_index = _ncvar_to_dict(ncvars['sweep_start_ray_index'])
+    #sweep_end_ray_index = _ncvar_to_dict(ncvars['sweep_end_ray_index'])
+
+    if 'target_scan_rate' in ncvars:
+        target_scan_rate = cfradial._ncvar_to_dict(ncvars['target_scan_rate'])
+    else:
+        target_scan_rate = None
+    if 'rays_are_indexed' in ncvars:
+        rays_are_indexed = cfradial._ncvar_to_dict(ncvars['rays_are_indexed'])
+    else:
+        rays_are_indexed = None
+    if 'ray_angle_res' in ncvars:
+        ray_angle_res = cfradial._ncvar_to_dict(ncvars['ray_angle_res'])
+    else:
+        ray_angle_res = None
+
+    # first sweep mode determines scan_type
+    # this module is specific to vertically-pointing data
+    mode = 'vertical_pointing'
+    scan_type = 'vpt'
+
+    # 4.8 Sensor pointing variables -> create attribute dictionaries
+    # this section also required some changes since the initial NetCDF did not
+    # contain any sensor pointing variables
+    azimuth = {
+        'data': 0.0 * np.ones(ncvars['time'].size, dtype=np.float32),
+        'long_name': 'Azimuth angle relative to true north',
+        'standard_name': 'beam_azimuth_angle',
+        'units': 'degrees',
+        'axis': 'radial_azimuth_angle',
+        'comment': 'Azimuth of antenna relative to true north',
+    }
+    elevation = {
+        'data': 90.0 * np.ones(ncvars['time'].size, dtype=np.float32),
+        'long_name': 'Elevation angle from horizontal plane',
+        'standard_name': 'beam_elevation_angle',
+        'units': 'degrees',
+        'axis': 'radial_elevation_coordinate',
+        'comment': 'Elevation of antenna relative to the horizontal plane',
+    }
+
+    #azimuth = _ncvar_to_dict(ncvars['azimuth'])
+    #elevation = _ncvar_to_dict(ncvars['elevation'])
+
+    if 'scan_rate' in ncvars:
+        scan_rate = cfradial._ncvar_to_dict(ncvars['scan_rate'])
+    else:
+        scan_rate = None
+
+    if 'antenna_transition' in ncvars:
+        antenna_transition = cfradial._ncvar_to_dict(
+            ncvars['antenna_transition'])
+    else:
+        antenna_transition = None
+
+    # 4.9 Moving platform geo-reference variables
```

Comment:
I think this section can be removed.  I don't know how ARM is specifying movement on shipborne zenith pointing radars but I doubt it follow the Cf/Radial convention.